src/main/drivers/timer_def.h

```diff
@@ -553,9 +553,9 @@
 #define DEF_TIM_DMA__BTCH_TIM5_CH3    D(1, 0, 6)
 #define DEF_TIM_DMA__BTCH_TIM5_CH4    D(1, 1, 6),D(1, 3, 6)
 
-#define DEF_TIM_DMA__BTCH_TIM8_CH1    D(2, 2, 7),D(2, 2, 0)
-#define DEF_TIM_DMA__BTCH_TIM8_CH2    D(2, 3, 7),D(2, 2, 0)
-#define DEF_TIM_DMA__BTCH_TIM8_CH3    D(2, 4, 7),D(2, 2, 0)
+#define DEF_TIM_DMA__BTCH_TIM8_CH1    D(2, 2, 0),D(2, 2, 7)
```

Comment:
What makes this reordering necessary?

Reply:
Make it uniform with F4, as this order is arbitrary and chosen by us anyway.
It was overlooked for some (just one?) targets that this order doesn't match F4, and hence Asgard32 F4 and F7 became different.

Reply:
Ah. Makes sense.